docs/talks/intro-api/talk.Rmd

```diff
@@ -0,0 +1,349 @@
+sympt = covidcast_signal(data_source = "fb-survey", 
+                         signal = "smoothed_hh_cmnty_cli",
+                         start_day = "2020-04-15", 
+                         geo_values = c(name_to_fips("Allegheny"),
+                                        name_to_fips("Fulton", state = "GA")))
+
+plot(sympt, plot_type = "line", range = range(sympt$value),
+     title = "% of people who know somebody with COVID symptoms")
+```
+
+# API Specification
+
+The COVIDcast API is based on HTTP GET queries and returns data in JSON form. The base URL is https://delphi.cmu.edu/epidata/api.php?source=covidcast
+
+<br>
+
+| Parameter | Description | Examples |
+| :-- | :-- | :-- |
+| `data_source` | data source | `doctor-visits` or `fb-survey` |
+| `signal` | signal derived from data source | `smoothed_cli` or `smoothed_adj_cli` |
+| `time_type` | temporal resolution of the signal | `day` or `week` |
+| `geo_type` | spatial resolution of the signal | `county`, `hrr`, `msa`, or `state` |
+| `time_values` | time units over which events happened | `20200406` or `20200406-20200410` |
+| `geo_value` | location codes, depending on `geo_type` | `*` for all, or `pa` for Pennsylvania |
+
+# Example: API Query
+
+Estimated % COVID-like illness on April 6, 2020 from the Facebook survey, in Allegheny County: https://delphi.cmu.edu/epidata/api.php?source=covidcast&data_source=fb-survey&signal=raw_cli&time_type=day&geo_type=county&time_values=20200406&geo_value=42003 
+
+```{r}
+library(jsonlite)
+res = readLines("https://delphi.cmu.edu/epidata/api.php?source=covidcast&data_source=fb-survey&signal=raw_cli&time_type=day&geo_type=county&time_values=20200406&geo_value=42003")
+prettify(res)
+```
+
+# API Documentation
+
+For full details, see the [API documentation site](https://cmu-delphi.github.io/delphi-epidata/api/covidcast.html). There you'll also find details on:
+
+- [Data sources and signals](https://cmu-delphi.github.io/delphi-epidata/api/covidcast_signals.html) 
+- [Geographic coding](https://cmu-delphi.github.io/delphi-epidata/api/covidcast_geography.html)
+- [COVIDcast metadata](https://cmu-delphi.github.io/delphi-epidata/api/covidcast_meta.html)
+
+# As Of, Issues, Lag
+
+By default the API returns the most recent data for each `time_value`. We also provide access to all previous versions of the data, using the following optional parameters:
+
+<br>
+
+| Parameter | To get data ... | Examples |
+| :-- | :-- | :-- |
+| `as_of` | as if we queried the API on a particular date | `20200406` |
+| `issues` | published at a particular date or date range | `20200406` or `20200406-20200410` |
+| `lag` | published a certain number of time units after events occured | `1` or `3` |
+
+# Data Revisions 
+
+Why would we need this? Because many data sources are subject to `r col("revisions")`:
+
+- Case and death counts frequently corrected/adjusted by authorities
+- Medical claims data can take days to be submitted and/or processed
```

Comment:
> Medical claims data can take days to be submitted and/or processed

DV continues to backfill for **weeks** -- we still get changes of +/- 5% at 45 days. Changing days to weeks here makes a stronger argument for revision tracking, and sets up slide 24 nicely.

Reply:
Thanks, fixed!